documentation/ame-guide/modules/ROOT/pages/modeling/connect-elements.adoc

```diff
@@ -14,27 +19,18 @@ image::connecting-elements.png[width=50%]
 
 * If only one direction is permitted, the arrow will be set respectively.
 * If both directions are permitted, the arrow will be set for the element selected first to the element selected second.
-* Tf a connection is not permitted, a respective warning will appear and the elements will not be connected.
-
-TIP: Alternatively, you can use the context menu (right mouse click) to access the action items.
-
-The larger the diagram, the harder it might get elements that are fare away.
+* If a connection is not permitted, a respective warning will appear and the elements will not be connected.
 
-In case you cannot connect two elements just the way you are most familiar with, try one of these procedures.
+The larger the diagram, the harder it might get to connect elements that are far away in the model. In that case, you can also use the alternative decribed bellow.
```

Comment:
```suggestion
The larger the diagram, the harder it might get to connect elements that are far away in the model. In that case, you can also use the alternative described bellow.
```